low-level/framework/sspl_ll_d

```diff
@@ -808,6 +809,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
+        logger.info(f"{traceback.print_exc()}")
```

Comment:
debug residual ? We discussed earlier also not to keep traceback

Reply:
Yes, sorry. Now its in debug level.